src/Spout/Common/Helper/FileSystemHelper.php

```diff
@@ -61,7 +61,7 @@ public function createFileWithContents($parentFolderPath, $fileName, $fileConten
         $filePath = $parentFolderPath . '/' . $fileName;
 
         $wasCreationSuccessful = file_put_contents($filePath, $fileContents);
-        if ($wasCreationSuccessful === false) {
+        if (false === $wasCreationSuccessful) {
```

Comment:
Is this a php-cs-fixer rule? I don't really like this order... I think it's harder to read this way

Reply:
Yes. I believe it is called "Yoda Style": https://en.wikipedia.org/wiki/Yoda_conditions.  See here: https://github.com/FriendsOfPHP/PHP-CS-Fixer

Maybe you can adjust the style settings to your liking? I personally always write boolean and === null expression this way. But I do not care, really...

Reply:
Oh I see. You actually need to pass it the actual config. I just updated it to disable yoda style : #472

Reply:
Also, you can run the fixer this way: `./vendor/bin/php-cs-fixer fix --config=.php_cs.dist`